packages/fuels-test-helpers/src/lib.rs

```diff
@@ -188,19 +190,26 @@ pub async fn setup_test_client(
     let config_with_coins: Value =
         serde_json::from_str(result.as_str()).expect("Failed to build config_with_coins JSON");
 
+    let transaction_parameters = serde_json::to_value(ConsensusParameters::DEFAULT).expect("Failed to build transaction_parameters JSON");
```

Comment:
Yes, default params are just another form of hard coding and don't respect what the node may actually be using.

Reply:
Looking closer at the context here, it seems like this is just setting the default params for the test node used in the harness? Since we're spawning a new node here, using default params for tests seems fine. Although I think the other comments from @segfault-magnet about using the actual ConsensusParams type instead of JSON is important though.

Reply:
That's true, this is in `setup_test_client()`. So it's 100% fine to use the defaults in this case.

Reply:
Now we have ``consensus_parameters_config: Option<ConsensusParameters>`` as parameter when starting node so we can configure it as we want.